app/components/pair-compare.jsx

```diff
@@ -0,0 +1,287 @@
+        }
+
+        setPointsIdxCounter(pointsIdxCounter + 2)
+
+    }
+
+    const handleStartOverButton = () => {
+        onDone({ valid: false, value: null })
+        setIdxRight(1)
+        setIdxLeft(0)
+        setPointsIdxCounter(1)
+        setSelectedPoint(null)
+    }
+
+    const isSelectionComplete = () => {
+        return pointsIdxCounter >= pointList.length
+    }
+
+    const nextIndex = idxLeft > idxRight ? idxLeft + 1 : idxRight + 1
+    const hiddenEmptyLeftPoint = <Point ref={hiddenLeftPointRef} className={classes.emptyPoint} />
+    const hiddenTransitioningLeftPoint = <Point ref={hiddenLeftPointRef} className={classes.emptyPoint} subject={pointList[nextIndex]?.subject} description={pointList[nextIndex]?.description} />
+    const hiddenEmptyRightPoint = <Point ref={hiddenRightPointRef} className={classes.emptyPoint} />
+    const hiddenTransitioningRightPoint = <Point ref={hiddenRightPointRef} className={classes.emptyPoint} subject={pointList[nextIndex]?.subject} description={pointList[nextIndex]?.description} />
+
+    return (
+        <div className={classes.container} {...otherProps}>
+
+            <div className={classes.mainPointContainer}>
+                <div className={classes.mainPointSubject}>{mainPoint.subject}</div>
+                <div className={classes.mainPointDescription}>{mainPoint.description}</div>
+            </div>
+
+            <span className={isSelectionComplete() ? classes.statusBadgeComplete : classes.statusBadge}>{`${pointsIdxCounter <= pointList.length ? pointsIdxCounter : pointList.length} out of ${pointList.length}`}</span>
+
+            <div className={classes.lowerContainer}>
+
+                <div className={classes.hiddenPointContainer}>
+                    {pointsIdxCounter < pointList.length &&
+                        <div className={classes.hiddenPoint}>{isRightTransitioning ? hiddenTransitioningLeftPoint : hiddenEmptyLeftPoint}</div>}
+                    {pointsIdxCounter < pointList.length &&
+                        <div className={classes.hiddenPoint}>{isLeftTransitioning ? hiddenTransitioningRightPoint : hiddenEmptyRightPoint}</div>}
+                </div>
+
+                <div className={classes.visiblePointsContainer}>
+                    {idxLeft < pointList.length &&
+                        <div className={classes.visiblePoint} ref={visibleLeftPointRef} onClick={handleLeftPointClick}>{<Point {...pointList[idxLeft]} />}</div>}
+                    {idxRight < pointList.length &&
+                        <div className={classes.visiblePoint} ref={visibleRightPointRef} onClick={handleRightPointClick}>{<Point {...pointList[idxRight]} />}</div>}
+                </div>
+
+                <div className={classes.buttonsContainer}>
+                    {
+                        !isSelectionComplete() ?
+                            <PrimaryButton className={classes.neitherButton} onClick={handleNeitherButton}>Neither</PrimaryButton>
```

Comment:
The secondary button should look exactly like you want (I think) so please don't apply classes.neitherButton or classes.startOverButton.   The idea is that these styles are in the Button and the button looks the same wherever it is.

Also, please use onDone instead of onClick.  onClick works but it's applied to the span around the button tag, rather than the button.  Also want consistency.